app/[contentType]/[slug]/page.tsx

```diff
@@ -174,16 +175,3 @@ function TechStack({ techStack }: { techStack: string[] }) {
     </li>
   );
 }
-
-export interface IContentData {
-  id: string;
-  contentHtml: string;
-  date: Date;
-  title: string;
-  previewImage?: string;
-  description?: string;
-  tags?: string[];
-  category?: string;
-  problem?: string;
-  techStack?: string[];
-}
```

Comment:
Moved to `content.ts` because I need the type in this file and importing from dynamic pages throws this ESLint error
```
Avoid imports from other pages in Next.js - this can cause errors with Next.js Statically Typed Links https://nextjs.org/docs/app/building-your-application/configuring/typescript#statically-typed-links

Instead, move anything you want to import to a non-page fileeslint[no-restricted-syntax](https://eslint.org/docs/latest/rules/no-restricted-syntax)
```